example-widget-mui/package.json

```diff
@@ -37,6 +36,9 @@
     "test": "jest --watch",
     "depcheck": "depcheck --ignores=@types/node --ignore-dirs=build",
     "translate": "i18next src/**/*.{ts,tsx}",
+    "pack": "echo \"Nothing to pack\"",
```

Comment:
Can you check if these scripts are needed? If not I would say we shouldn't include them.

Reply:
I will remove them - these run when doing the `npm pack` stage of publishing a new release and I checked that removing them doesn't effect the process.